src/natcap/invest/validation.py

```diff
@@ -522,7 +526,7 @@ def check_csv(filepath, required_fields=None, excel_ok=False):
         # use sep=None, engine='python' to infer what the separator is
         dataframe = pandas.read_csv(
             filepath, sep=None, engine='python', encoding=encoding)
-    except Exception:
+    except Exception as e:
```

Comment:
Was this a linting change?

Reply:
no I did that when I was going to use an exception instead of warning message. Forgot to change it back 🙃